torchtune/models/clip/_component_builders.py

```diff
@@ -19,6 +19,7 @@ def clip_vision_encoder(
     output_cls_projection: bool = False,
     max_num_tiles: int = 4,
     in_channels: int = 3,
+    hidden_act: torch.nn.Module = torch.nn.SiLU(),
```

Comment:
nit: maybe call this `intermediate_activation` to be more clear which activation in the transformer it is changing?